src/pages/index.js

```diff
@@ -38,29 +49,38 @@ const profileDescriptionInput = document.querySelector(
 //Add modal
 const addNewCardButton = document.querySelector(".profile__add-button");
 const addCardModal = document.querySelector("#profile-add-modal");
+const EditAvatarModal = document.querySelector("#profile-avatar-modal");
+
+//Delete Modal
+//const cardDeleteModal = document.querySelector("#modal-delete-card");
+//const cardDeleteButton = document.querySelector(".card__delete-button");
 
 /*FormValidator.js*/
 
 const editFormElement = profileEditModal.querySelector(".modal__form");
 const addFormElement = addCardModal.querySelector(".modal__form");
+const avatarFormElement = EditAvatarModal.querySelector(".modal__form");
 
 const editFormValidator = new FormValidator(settings, editFormElement);
 editFormValidator.enableValidation();
 
 const addFormValidator = new FormValidator(settings, addFormElement);
 addFormValidator.enableValidation();
 
+const editAvatarValidator = new FormValidator(settings, avatarFormElement);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object: `formValidators`.  Then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // Here you get the name of the form (if you don’t have it then you need to add it into each form in `index.html` first)
    const formName = formElement.getAttribute('name')

   // Here you store the validator using the `name` of the form
    formValidators[formName] = validator;
    validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```